src/tasks/tasks.py

```diff
@@ -75,13 +76,21 @@ def process_single_pair_task_split(
     Returns:
         - instances (Iterable[Instance]): an iterable of AllenNLP Instances with fields
     '''
+    # check here if using bert to avoid passing model info to tasks
+    bert_pair = any([idx_name == BERT_INDEXER_NAME for idx_name in indexers])
```

Comment:
Since this is an exact match now, what about just `BERT_INDEXER_NAME in indexers` ?

Also: `is_bert_pair` for consistency with `is_pair`? (or: `is_using_bert` since I think this is true even for non-pair tasks)?

Reply:
:+1:, will change